Targets/JavaScriptCore/fuzzbuild.sh

```diff
@@ -18,6 +18,10 @@ export WEBKIT_OUTPUTDIR=FuzzBuild
 
 if [ "$(uname)" == "Linux" ]; then
     ./Tools/Scripts/build-jsc --jsc-only --debug --cmakeargs="-DENABLE_STATIC_JSC=ON -DCMAKE_C_COMPILER='/usr/bin/clang' -DCMAKE_CXX_COMPILER='/usr/bin/clang++' -DCMAKE_CXX_FLAGS='-fsanitize-coverage=trace-pc-guard -O3 -lrt'"
+elif [ "$(uname)" == "Darwin" ]; then
+    unset WEBKIT_OUTPUTDIR # remove the outputdir env on macOS for the build without cmake will be in the Source directory(maybe the build-jsc have problem)
```

Comment:
I'm not sure I understand this. So if that environment variable exist, the build artifacts will be placed in Source/? Or the other way around?

Reply:
In the past, I have just set the sub project to build with xcode settings that reletive to source folder directly within opening the JavascriptCore proj

With the debug of my env, maybe my wrong settings for the sub xcode project make the problem, I will try with that again and check what the workflow of build with the perl script and Makefile.

Try with the export WEBKIT_OUTPUTDIR to FuzzBuild.
And the build script just lead to this pic which means if this env exist, the artifacts will just in the `Source/` folder
I think this should be the problem caused by the Makefile
![image](https://github.com/googleprojectzero/fuzzilli/assets/49717931/221eee95-a0df-4338-bdc4-292fff4a6258)